pcdsdevices/mpodApalis.py

```diff
@@ -0,0 +1,248 @@
+    c4 = Cpt(MPODApalisChannel, ":C4")
+    c5 = Cpt(MPODApalisChannel, ":C5")
+    c6 = Cpt(MPODApalisChannel, ":C6")
+    c7 = Cpt(MPODApalisChannel, ":C7")
+
+
+class MPODApalisModule16Channel(MPODApalisModule):
+
+    """
+    MPODApalis 16 channel Module Object.
+
+    Parameters
+    ----------
+    card_prefix : str
+        The EPICS base of the MPOD Module. `TMO:MPOD:01:M6
+    name : str
+       A name to refer to the device.
+    """
+
+    c0 = Cpt(MPODApalisChannel, ":C0")
+    c1 = Cpt(MPODApalisChannel, ":C1")
+    c2 = Cpt(MPODApalisChannel, ":C2")
+    c3 = Cpt(MPODApalisChannel, ":C3")
+    c4 = Cpt(MPODApalisChannel, ":C4")
+    c5 = Cpt(MPODApalisChannel, ":C5")
+    c6 = Cpt(MPODApalisChannel, ":C6")
+    c7 = Cpt(MPODApalisChannel, ":C7")
+    c8 = Cpt(MPODApalisChannel, ":C8")
+    c9 = Cpt(MPODApalisChannel, ":C9")
+    c10 = Cpt(MPODApalisChannel, ":C10")
+    c11 = Cpt(MPODApalisChannel, ":C11")
+    c12 = Cpt(MPODApalisChannel, ":C12")
+    c13 = Cpt(MPODApalisChannel, ":C13")
+    c14 = Cpt(MPODApalisChannel, ":C14")
+    c15 = Cpt(MPODApalisChannel, ":C15")
+
+
+class MPODApalisModule24Channel(MPODApalisModule):
+
+    """
+    MPODApalis 24 channel Module Object.
+
+    Parameters
+    ----------
+    card_prefix : str
+        The EPICS base of the MPOD Module. `TMO:MPOD:01:M6
+    name : str
+       A name to refer to the device.
+    """
+
+    c0 = Cpt(MPODApalisChannel, ":C0")
+    c1 = Cpt(MPODApalisChannel, ":C1")
+    c2 = Cpt(MPODApalisChannel, ":C2")
+    c3 = Cpt(MPODApalisChannel, ":C3")
+    c4 = Cpt(MPODApalisChannel, ":C4")
+    c5 = Cpt(MPODApalisChannel, ":C5")
+    c6 = Cpt(MPODApalisChannel, ":C6")
+    c7 = Cpt(MPODApalisChannel, ":C7")
+    c8 = Cpt(MPODApalisChannel, ":C8")
+    c9 = Cpt(MPODApalisChannel, ":C9")
+    c10 = Cpt(MPODApalisChannel, ":C10")
+    c11 = Cpt(MPODApalisChannel, ":C11")
+    c12 = Cpt(MPODApalisChannel, ":C12")
+    c13 = Cpt(MPODApalisChannel, ":C13")
+    c14 = Cpt(MPODApalisChannel, ":C14")
+    c15 = Cpt(MPODApalisChannel, ":C15")
+    c16 = Cpt(MPODApalisChannel, ":C16")
+    c17 = Cpt(MPODApalisChannel, ":C17")
+    c18 = Cpt(MPODApalisChannel, ":C18")
+    c19 = Cpt(MPODApalisChannel, ":C19")
+    c20 = Cpt(MPODApalisChannel, ":C20")
+    c21 = Cpt(MPODApalisChannel, ":C21")
+    c22 = Cpt(MPODApalisChannel, ":C22")
+    c23 = Cpt(MPODApalisChannel, ":C23")
+
+
+class MPODApalisCrate(BaseInterface, Device):
+
+    """
+    MPODApalis Crate Object.
+
+    Parameters
+    ----------
+    card_prefix : str
+        The EPICS base of the MPOD Crate. `TMO:MPOD:01
+    name : str
+       A name to refer to the device.
+    """
+
+    power = Cpt(EpicsSignal, ':Crate:PowerOn',
+                kind='normal', string=True,
+                doc='Crate power status and control')
+
+    tab_component_names = True
+    tab_whitelist = ['power']
+
+    def power_cycle(self):
+        """
+       Function used to power cycle the MPOD crate
+        """
```

Comment:
Misaligned docstring
```suggestion
        """
        Function used to power cycle the MPOD crate
        """
```